### PR TITLE
Fixes cults converting things they shouldn't be able to

### PR DESCRIPTION
**Pull request by ChangelingRain**
Including silicons, drones, bots, and guardians with a noncultist owner.

### CODE REVIEW

**Comment by Robustin** [comment]
Booo, sillicon conversion is a FEATURE, they have souls too!

File: code/game/gamemodes/cult/cult.dm [hunk 1]
```diff
@@ -21,8 +21,15 @@
 		return 0
 	if(isloyal(mind.current))
 		return 0
-	if (ticker.mode.name == "cult")		//redundant?
-		if(is_sacrifice_target(mind))	return 0
+	if(issilicon(mind.current) || isbot(mind.current) || isdrone(mind.current))
+		return 0 //can't convert machines, that's ratvar's thing
```
Comment (Man-Shroom-) [inline code/game/gamemodes/cult/cult.dm]: HELLO IS THIS SOME LORE

I AM SUMMONED

**Comment by ChangelingRain** [comment]
Silicon conversion is specifically ratvar's thing, not some random blood god. Robots don't even have blood!

**Comment by Man-Shroom-** [comment]
this lore is approved by me, the offical /tg/station head of all lore™ and backstories™ as promoted by our glorious and esteemed administration™

**Comment by xxalpha** [comment]
There is nothing wrong with culting cyborgs.

Needs a 'removes fun' label.

**Comment by Aloraydrel** [comment]
#notallcyborgs

**Comment by PKPenguin321** [comment]
drones, sure, but the others should stay in. "muh lore!!!" is a bad reason to remove them

**Comment by Durkel** [comment]
I'd argue balance wise it's bad to convert borgs, considering how easy a flash can disable them coupled with the instant all access,door control, and the fact it would fuck with laws and such. We don't let borgs be any other antag, why make a exception for this?

**Comment by InsaneHyena** [comment]
Fun removal. Removing a fun, never used feature that was only just discovered in a controlled enviroment because of "muh lores" is bad.

**Comment by Eaglendia** [comment]
A cult cyborg still has laws, I believe, unless the conversion explicitly removes them. In this case the borg can't really help the cult, as all of their objectives constitute harm (assuming Asimov). Likewise, does a cult borg answer first to its AI or to the other cultists? And what if it had a law mandating the death of cultists?

And what happens if a holoparasite is converted with an unconverted master? Are they technically enemies?

These situations are probably a lot more fun to think about than they are to play.

**Comment by Anonmare** [comment]
I'll have to check in Thunderdome if a converted silicon has it's lawset intact or not. 
If they do then that's an exploit for an antag HUD.

**Comment by InsaneHyena** [comment]
Converting cyborgs makes them cult. Cultists are antags. Antag silicons never have to follow any of their laws.

**Comment by GunHog** [comment]
Antag silicons *have irremovable laws stating that they are antags*. Laws are everything to a Silicon, right below admin and server rules. Having antag status without matching laws is a point of confusion. That being said, it should be resolved that either A: Silicons cannot be converted, or B: Converted silicons gain an antag law 0. 

(I was going to include an example about converting Syndieborgs to Asimov to make them station controlled, but when I tested this for proof, I encountered runtimes preventing law boards working on cyborgs)

**Comment by Anonmare** [comment]
@InsaneHyena  That's not how it works you silly little man

**Comment by PKPenguin321** [comment]
>only tag on the PR is a self-applied "Fix" tag

that's a grimy move

**Comment by KorPhaeron** [comment]
If you want to "convert" borgs give them new laws, it's just confusing otherwise

**Comment by PKPenguin321** [comment]
bots at the very least should have been exempt

**Pull request merged**